source-reminder/lambda_function.py

```diff
@@ -0,0 +1,146 @@
+import gspread
+from oauth2client.service_account import ServiceAccountCredentials
+import requests
+from datetime import date, datetime, timedelta
+import logging
+import os
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
+
+scope = ['https://spreadsheets.google.com/feeds', 'https://www.googleapis.com/auth/drive', 'https://www.googleapis.com/auth/admin.directory.user']
+API_ENDPOINT = 'https://www.texastribune.org/api/v2/articles/'
+credentials = ServiceAccountCredentials.from_json_keyfile_name('credentials.json', scope)  # Add credentials to the service account.
```

Comment:
When we move this into Lambda, we'll have to figure out how to best protect/load this `credentials.json` file, since we can't version-control it.  Do you think these values could be loaded in as an environment variable?  If not one option could be to look into AWS Secrets Manager to see if we can load it through there.

Reply:
( also happy to help look into how to do this when we move to lambda )